cryptoki/src/session/message_decryption.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright 2025 Contributors to the Parsec project.
 // SPDX-License-Identifier: Apache-2.0
-//! Encrypting data
+//! Decrypting messages
```

Comment:
:detective:

Reply:
Sorry, did not write it right the first time :)